src/axom/quest/examples/quest_distributed_distance_query_example.cpp

```diff
@@ -848,15 +1004,31 @@ int main(int argc, char** argv)
     exit(retval);
   }
 
+  // Issue warning about result-checking requiring good resolution.
+  if(params.checkResults &&
+     (params.circlePoints < 100 ||
+      (params.randomSpacing && params.circlePoints < 500)))
```

Comment:
Good point!  I've been using r=1, but those values should be scaled by r.  And the warning doesn't apply for uniform spacing, because we now calculate the correct slack to give it.